QKSMS/src/main/java/com/moez/QKSMS/ui/messagelist/MessageListFragment.java

```diff
@@ -474,7 +475,6 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 makeCall();
                 return true;
 
-            case R.id.menu_uw_convo_private_text:
```

Comment:
Sorry, my bad. Yes of course the _text is for the items in menu. The SetPrivate with icon gets pushed to the menu on my device if showasaction is set to ifRoom. And if showasaction is set to "always" , its icon appears in the top header but then the Name of the Sender disappears due to lack of space. So I will just show SetPrivate with _text in the menu.